### ISSUE TITLE
Publish to docker image action is broken

### ISSUE THREAD

**Issue by EirikKolas** [vortexntnu/deprecated-vortex-cv#130]
**Time estimate:** 1 hour
**Deadline:**  

**Description of task:** Upload to Docker github action fails
- Action currently fails on installing one of the python dependencies.
- It will probably also fail because after #116 gets merged, we will have a mix of python2 and python3 packages
- Should move OpenCV part of the dockerfile to separate repo to reduce build time and for code quality (separate issue?)

**Comment by EirikKolas** [comment]
What do we do when we have a mix of noetic and melodic packages...
